server/sonar-server/src/main/java/org/sonar/server/computation/issue/SourceLinesCache.java

```diff
@@ -69,7 +70,8 @@ void init(String fileUuid) {
   public String lineAuthor(int lineId) {
     loadIfNeeded();
     if (lineId <= authors.size()) {
-      return authors.get(lineId - 1);
+      String author = authors.get(lineId - 1);
```

Comment:
Why go the null way when we can return an empty string ? No NPE, we can do a var.isEmpty(), lass things to worry about.